saleor/graphql/schema.graphql

```diff
@@ -995,7 +995,7 @@ enum CheckoutErrorCode {
   ZERO_QUANTITY
   MISSING_CHANNEL_SLUG
   CHANNEL_INACTIVE
-  UNAVAILABLE_VARIANT_IN_CHANNEL
+  NOT_AVAILABLE_IN_CHANNEL
```

Comment:
IMO breaking changes at least for someone who uses alpha releases.

Reply:
We need to check with our frontend team if this field is already used and if so it should be changed in the frontend, cc @jwm0.